.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I am not sure that we want to commit the vscode file?
Shouldn't we be agnostic from the IDE?

Reply:
removed in latest commit